spec/models/artist_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Artist, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+    describe 'Artist' do
+     let(:attributes) {{
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.